workflows/magmap.nf

```diff
@@ -182,6 +191,41 @@ workflow MAGMAP {
             .map { [ it[0], it[1], it[2], [] ] }
     }
 
+    //
+    // MODULE: Kraken2
+    //
+    if ( ! skip_kraken2 ) {
+        if (!kraken2_db) {
+        // Choose database based on type
+            db_name = kraken2_db_type
```

Comment:
```suggestion
```

Reply:
params.kraken2_db_type is still used in KRAKEN2_DOWNLOAD.nf module. should I remove it from there then?

Reply:
this is the way the module works.
```
input:
    val db_name
    val db_url
    
    output:
    path "${db_name}/" , emit: db_dir
    path "versions.yml", emit: versions
    
    script:
    """
    # Download the database
    wget -O ${db_name}.tar.gz ${db_url}
    
    # Create directory and extract
    mkdir -p ${db_name}
    tar -xzf ${db_name}.tar.gz -C ${db_name}
    
    # Clean up archive
    rm ${db_name}.tar.gz
```

`--kraken2_db_type`is defining the name for the database folder in the end